src/default-controls.js

```diff
@@ -39,15 +39,64 @@ export class NextButton extends React.Component {
   constructor() {
     super(...arguments);
     this.handleClick = this.handleClick.bind(this);
+    this.nextButtonDisable = this.nextButtonDisabled.bind(this);
   }
   handleClick(event) {
     event.preventDefault();
     this.props.nextSlide();
   }
+
+  nextButtonDisabled(params) {
```

Comment:
I would actually rename this to something like `disableNextButton` - the convention of naming these "action" methods with the verb first helps the reader/future dev understand what the method is doing, while still being able to use the function invocation as a boolean if needed.

Reply:
My only nit with `disableNextButton` (but feel free to ignore!!!) is that it could be read that one could invoke `this.disableNextButton()` and the next button would in fact be disabled (or enabled), but it won't.  This method only returns a boolean which could then be used to disable/enable the next button.  Maybe it should be called `shouldDisableNextButton`, to clearly read that it is only returning a flag to be used and not actually doing the enabling/disabling itself, but that could be overkill in naming :)

Reply:
Ah true - `disableNextButton` gives the impression that this method is in fact disabling the button, so I will redact my comment...it would make more sense to keep it as it is in that case! this naming thing is hard :)

Reply:
Thanks!